app/sequencescape/sequencescape/api/v2/request_metadata.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Represents a request metadatum in Limber via the Sequencescape API
+class Sequencescape::Api::V2::RequestMetadata < Sequencescape::Api::V2::Base
```

Comment:
metadatum in the comment is confusing I think. RequestMetadata is set to be singular for this resource (inflections, uncountable). I would avoid using both forms in the same context.

Reply:
I think I changed the class name and forgot the comment. Will fix.